res/database/Default/user/scr0llbaer_constr_d2v_tv_series.xml

```diff
@@ -0,0 +1,6397 @@
+					<title_original>
+						<de>Der große Bluff</de>
+						<en>The Wheeler Dealer</en>
+					</title_original>
+					<description>
+						<de>Während ein Großteil der Familie Jocks erste Frau in einem Sanatorium in Colorado besucht, nutzt J.R. die Zeit, um ein noch größeres Ölgeschäft durch die nun einträglichen asiatischen Bohrungen anzubahnen.</de>
+						<en>With most of the family visiting Jock's first wife in a Colorado sanitarium, J.R. uses the time to try to set up an even bigger oil deal with the now profitable Asian wells.</en>
+					</description>
+				</programme>
+				<programme id="00631347-7cdd-4af1-a977-b6f03fc139fa" product="2" licence_type="2" imdb_id="tt0553252" creator="9551">
+					<title>
+						<de>Ein geteiltes Haus</de>
+						<en>Reckoning</en>
+					</title>
+					<title_original>
+						<de>Abrechnung</de>
+						<en>A House Divided</en>
+					</title_original>
+					<description>
+						<de>J.R. verkauft die asiatischen Ölpachten, bevor die Nachricht von der Verstaatlichung der Ölfelder bekannt wird, und verursacht damit großes Aufsehen. Es gibt viele Verdächtige, als er beim Arbeiten spät im Büro niedergeschossen wird.</de>
+						<en>J.R.'s selling of the Asian oil leases before news of the fields' nationalization is made public creates havoc. Suspects are aplenty when J.R. while working late at the office, is shot.</en>
+					</description>
+				</programme>
+			</children>
+		</programme>
+		<programme id="ab23ad17-1e55-4ed5-b15f-52a0f5f6882c" product="2" licence_type="3" creator="9551" imdb_id="tt0077000" comment="8 in-game episodes representing 23 real ones">
+			<title>
+				<de>Der Texas-Clan (Staffel 4)</de>
+				<en>North Texas (Season 4)</en>
+			</title>
+			<title_original>
+				<de>Dallas (Staffel 4)</de>
+				<en>Dallas (Season 4)</en>
+			</title_original>
+			<description>
+				<de>Mit J.R.'s Gier wachsen auch die Risiken, die ihn zu Fall zu bringen drohen. Das Auftauchen seines Halbbruders führt zu neuen Komplexitäten, Konflikten und Enthüllungen. Ein tragisches Ereignis erschüttert Southfork.</de>
+				<en>As J.R.'s greed grows, so do the risks, threatening to bring him down. The emergence of his half-brother leads to new complexities, conflicts and revelations. A tragic event rocks Southfork.</en>
+			</description>
+			<staff>
+				<member index="0" function="2">0acdf48f-147d-46e6-8d71-9f3d9562841c</member>
+				<member index="1" function="2">eda99ee8-1e4e-4e58-b424-7e76b7a73df2</member>
+				<member index="2" function="2">958aff9d-37e7-42f5-8d8f-6f0e7f0780eb</member>
+				<member index="3" function="2">56d5b70e-6013-405f-82ad-9d5c97e2a2c3</member>
+				<member index="4" function="2">68e3fa59-30f3-4d65-8382-dcb54c51a116</member>
+				<member index="5" function="2">9e4ab7a6-a028-4f57-97f0-ec7207e696a8</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" />
+			<data country="USA" distribution="2" maingenre="7" subgenre="15" flags="0" blocks="1" price_mod="0.75" />
```

Comment:
Yes I thought about this too and we discussed this on discord that we could use triggers to "unlock" next seasons, also for movie series (though with series it's not clear where to apply the trigger. First episode, really? Last episode? Then player could 'cheat' etc). On the other hand there's already been a franchise feature in discussion for a long time, so these things could be part of that. Only first season or first part of a movie series is available at first, you buy the license, you get the option to adopt the whole franchise, i.e. you get a notification about buying the next part / season (with a little rebate), if you don't want it or can't afford it, it will be available for all players in the film agency.

Reply:
We could consider adding a trigger which "cumulates". The effect would happen on "first broadcast" of an episode (so only once). Each executed trigger adds to a value. And somewhere you check if the value exceeds a specified limit.

Simpler of course (?) would be a trigger/effect which happens on the series, not the episodes - so a series header effect which checks if all episodes were run.

Nonetheless nittka's suggestion of triggering availability with "episode 1" is already good enough - as the basic idea is to simply not being able to buy season 2 before season 1. Yes, it won't help against people not airing "episode 1" - and thus it requires the same effect on all episodes (so each of them would enable season 2...)

Reply:
Being lazy, I'd like to keep things simple if there is a 80% solution available. I consider buying a series and then not sending the first episode an absolute border case - just ignore it.
I would not count on the franchise feature being addressed soon. The effects are a ready to hand.

Reply:
As it is stated in the docu, this feature (effects for programmes) doesn't seem to be supported yet (last release version). Also the syntax isn't clear yet (as the example in the docu for `news` entries is news-specific).  Nevertheless I've applied the effect to the series 08d9d191-bdc5-477a-904d-85a19236b37a episode 1 with how I *think* the syntax will look like to test in the future.

Reply:
see #1059 

proposed syntax

<effect trigger="broadcastFirstTimeDone" type="modifyProgrammeAvailability" enable="1" guid="421850be-e53f-40ec-b9aa-04e69f85aa81" />

trigger "happen" does not make much sense for programmes (happen even if not broadcasted), type - specify what should be changed

Reply:
I can't find an example. So I applied this syntax now (guessing from the new code):

`<effect type="modifyAvailability" enable="1" programmeId="421850be-e53f-40ec-b9aa-04e69f85aa81" />`

(Which makes the last release version (not a compiled newer version) crash after starting a new game when reading the xml files though. My previous attempt didn't make it crash although it was "wrong" with unknown tags).

Reply:
The documentation has been updated. The PR for programme effecs is merged, for scripts it is still under review, but the syntax would be the same.

Reply:
applied the effects to "all" entries with (I hope) the correct syntax.